### PR TITLE
Update dosbox-x from 0.83.13 to 0.83.14

### PR DESCRIPTION
**Pull request by tomyun**
- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

---

- version bump
- zap added

### CODE REVIEW

File: Casks/dosbox-x.rb [hunk 1]
```diff
@@ -34,4 +34,8 @@
   homepage "https://dosbox-x.com/"
 
   app "dosbox-x/dosbox-x.app"
+
+  zap trash: [
+    "~/Library/Preferences/com.dosbox-x.plist",
+  ]
```
Comment (cho-m) [inline Casks/dosbox-x.rb:40]: For single trash path, the style preference is a single line:
```suggestion
  zap trash: "~/Library/Preferences/com.dosbox-x.plist"
```

Comment (tomyun) [reply Casks/dosbox-x.rb]: updated

**Review by cho-m — DISMISSED** [review]
Thanks, @tomyun

File: Casks/dosbox-x.rb (hunk 1 above)
Comment (tomyun) [reply Casks/dosbox-x.rb]: actually just found another file for zap. list updated again.

**Review by cho-m — APPROVED** [review]

**Pull request merged**